### ISSUE TITLE
Error with "stress_qualifier" sensor enabled - type str doesn't define round method

### ISSUE THREAD

**Issue by Broekman** [cyberjunky/home-assistant-garmin_connect#54]
Just enabled all sensors again to see what was coming in, and caught one error with the sensor "stress_qualifier" enabled.
- All other sensors are enabled as well and don't give errors.
- Version: 0.2.3
- Looks like strings default to that round(x, 2) return in sensor.py.

```
Error adding entities for domain sensor with platform garmin_connect
Error while setting up garmin_connect platform for sensor

2023-01-04 19:20:42.827 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up garmin_connect platform for sensor
Traceback (most recent call last):
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity_platform.py", line 289, in _async_setup_platform
await asyncio.gather(*pending)
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity_platform.py", line 428, in async_add_entities
await asyncio.gather(*tasks)
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity_platform.py", line 692, in _async_add_entity
await entity.add_to_platform_finish()
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity.py", line 789, in add_to_platform_finish
self.async_write_ha_state()
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity.py", line 545, in async_write_ha_state
self._async_write_ha_state()
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity.py", line 583, in _async_write_ha_state
state = self._stringify_state(available)
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/helpers/entity.py", line 551, in _stringify_state
if (state := self.state) is None:
File "/usr/share/webapps/homeassistant/lib/python3.10/site-packages/homeassistant/components/sensor/__init__.py", line 680, in state
value = self.native_value
File "/home/homeassistant/.homeassistant/custom_components/garmin_connect/sensor.py", line 127, in native_value
return round(value, 2)
TypeError: type str doesn't define __round__ method
```

**All OK for the other sensors when stress_qualifier is disabled:**
```
2023-01-04 19:25:51.367 DEBUG (MainThread) [custom_components.garmin_connect] Finished fetching garmin_connect data in 0.803 seconds (success: True)
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: totalSteps, Total Steps, steps, mdi:walk, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: dailyStepGoal, Daily Step Goal, steps, mdi:walk, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: totalKilocalories, Total KiloCalories, kcal, mdi:food, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: activeKilocalories, Active KiloCalories, kcal, mdi:food, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bmrKilocalories, BMR KiloCalories, kcal, mdi:food, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: consumedKilocalories, Consumed KiloCalories, kcal, mdi:food, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: burnedKilocalories, Burned KiloCalories, kcal, mdi:food, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: remainingKilocalories, Remaining KiloCalories, kcal, mdi:food, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: netRemainingKilocalories, Net Remaining KiloCalories, kcal, mdi:food, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: netCalorieGoal, Net Calorie Goal, kcal, mdi:food, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: totalDistanceMeters, Total Distance Mtr, m, mdi:walk, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: wellnessStartTimeLocal, Wellness Start Time, None, mdi:clock, timestamp, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: wellnessEndTimeLocal, Wellness End Time, None, mdi:clock, timestamp, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: wellnessDescription, Wellness Description, , mdi:clock, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: wellnessDistanceMeters, Wellness Distance Mtr, m, mdi:walk, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: wellnessActiveKilocalories, Wellness Active KiloCalories, kcal, mdi:food, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: wellnessKilocalories, Wellness KiloCalories, kcal, mdi:food, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: highlyActiveSeconds, Highly Active Time, min, mdi:fire, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: activeSeconds, Active Time, min, mdi:fire, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: sedentarySeconds, Sedentary Time, min, mdi:seat, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: sleepingSeconds, Sleeping Time, min, mdi:sleep, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: measurableAwakeDuration, Awake Duration, min, mdi:sleep, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: measurableAsleepDuration, Sleep Duration, min, mdi:sleep, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: floorsAscendedInMeters, Floors Ascended Mtr, m, mdi:stairs, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: floorsDescendedInMeters, Floors Descended Mtr, m, mdi:stairs, None, False
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: floorsAscended, Floors Ascended, floors, mdi:stairs, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: floorsDescended, Floors Descended, floors, mdi:stairs, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: userFloorsAscendedGoal, Floors Ascended Goal, floors, mdi:stairs, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: minHeartRate, Min Heart Rate, bpm, mdi:heart-pulse, None, True
2023-01-04 19:25:51.369 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: maxHeartRate, Max Heart Rate, bpm, mdi:heart-pulse, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: restingHeartRate, Resting Heart Rate, bpm, mdi:heart-pulse, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: minAvgHeartRate, Min Avg Heart Rate, bpm, mdi:heart-pulse, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: maxAvgHeartRate, Max Avg Heart Rate, bpm, mdi:heart-pulse, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: abnormalHeartRateAlertsCount, Abnormal HR Counts, None, mdi:heart-pulse, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: lastSevenDaysAvgRestingHeartRate, Last 7 Days Avg Heart Rate, bpm, mdi:heart-pulse, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: averageStressLevel, Avg Stress Level, lvl, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: maxStressLevel, Max Stress Level, lvl, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: stressQualifier, Stress Qualifier, None, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: stressDuration, Stress Duration, min, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: restStressDuration, Rest Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: activityStressDuration, Activity Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: uncategorizedStressDuration, Uncat. Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: totalStressDuration, Total Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: lowStressDuration, Low Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: mediumStressDuration, Medium Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: highStressDuration, High Stress Duration, min, mdi:flash-alert, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: stressPercentage, Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: restStressPercentage, Rest Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: activityStressPercentage, Activity Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: uncategorizedStressPercentage, Uncat. Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: lowStressPercentage, Low Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: mediumStressPercentage, Medium Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: highStressPercentage, High Stress Percentage, %, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: moderateIntensityMinutes, Moderate Intensity, min, mdi:flash-alert, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: vigorousIntensityMinutes, Vigorous Intensity, min, mdi:run-fast, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: intensityMinutesGoal, Intensity Goal, min, mdi:run-fast, None, False
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyBatteryChargedValue, Body Battery Charged, %, mdi:battery-charging-100, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyBatteryDrainedValue, Body Battery Drained, %, mdi:battery-alert-variant-outline, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyBatteryHighestValue, Body Battery Highest, %, mdi:battery-heart, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyBatteryLowestValue, Body Battery Lowest, %, mdi:battery-heart-outline, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyBatteryMostRecentValue, Body Battery Most Recent, %, mdi:battery-positive, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: averageSpo2, Average SPO2, %, mdi:diabetes, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: lowestSpo2, Lowest SPO2, %, mdi:diabetes, None, True
2023-01-04 19:25:51.370 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: latestSpo2, Latest SPO2, %, mdi:diabetes, None, True
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: latestSpo2ReadingTimeLocal, Latest SPO2 Time, None, mdi:diabetes, timestamp, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: averageMonitoringEnvironmentAltitude, Average Altitude, %, mdi:image-filter-hdr, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: highestRespirationValue, Highest Respiration, brpm, mdi:progress-clock, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: lowestRespirationValue, Lowest Respiration, brpm, mdi:progress-clock, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: latestRespirationValue, Latest Respiration, brpm, mdi:progress-clock, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: latestRespirationTimeGMT, Latest Respiration Update, None, mdi:progress-clock, timestamp, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: weight, Weight, kg, mdi:weight-kilogram, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bmi, BMI, bmi, mdi:food, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyFat, Body Fat, %, mdi:food, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: bodyWater, Body Water, %, mdi:water-percent, None, False
2023-01-04 19:25:51.371 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: boneMass, Bone Mass, kg, mdi:bone, None, False
2023-01-04 19:25:51.372 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: muscleMass, Muscle Mass, kg, mdi:dumbbell, None, False
2023-01-04 19:25:51.372 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: physiqueRating, Physique Rating, None, mdi:numeric, None, False
2023-01-04 19:25:51.372 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: visceralFat, Visceral Fat, %, mdi:food, None, False
2023-01-04 19:25:51.372 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: metabolicAge, Metabolic Age, y, mdi:calendar-heart, None, False
2023-01-04 19:25:51.372 DEBUG (MainThread) [custom_components.garmin_connect.sensor] Registering entity: nextAlarm, Next Alarm Time, None, mdi:alarm, None, True
```

**Comment by cyberjunky** [comment]
Thanks! I fixed that one too in version 0.2.5, a bit shocked by the value I have, but feeling a bit ill for a day now, so this matches.. ;-)

![image](https://user-images.githubusercontent.com/5447161/210630677-af1877c6-95ed-489f-868a-556848bc88e8.png)

**Comment by Broekman** [comment]
haha yes, I for some reason have the same. All is working now, thanks!

Closing the ticket 👍
